Guide agents to keep PR descriptions self-contained (#1351)
ai2cm/ace: AGENTS.md

When making PRs, use the template that exists in the repo under .github/pull_request_template.md, if one exists.

> The PR description becomes the squash-and-merge commit message, so keep it self-contained and describe the final state of the change from main. Keep PR-process and agent-iteration content out of it: commit-by-commit narratives (stale after squash), "review response" notes, stacking/rebase instructions, and review attributions belong in PR comments, not the description. Prefer posting such context as a PR comment rather than folding it into the description.

Author: ```suggestion
The PR description becomes the squash-and-merge commit message, so keep it self-contained and describe the final state of the change from main. PR-process information belongs in PR comments, not the description.
```
